model/validation.go

```diff
@@ -235,9 +235,9 @@ func ValidateFQDN(fqdn string) error {
 
 // ValidateIstioService checks for validity of a service reference
 func ValidateIstioService(svc *proxyconfig.IstioService) (errs error) {
-	if svc.Name == "" {
-		errs = multierror.Append(errs, errors.New("name is mandatory for a service reference"))
-	} else if !IsDNS1123Label(svc.Name) {
+	if svc.Name == "" && svc.Service == "" {
```

Comment:
check that if service is specified then name, namespace, domain are not.
check that service is FQDN/wildcard mask (see egress rule validation)